examples/surf-web.rs

```diff
@@ -1,15 +1,16 @@
 use async_std::task;
 
-fn main() -> Result<(), surf::Exception> {
+fn main() -> Result<(), http_types::Error> {
```

Comment:
`surf::Error` should be able to be used here; I don't think we manually need to pull in `http_types`

Reply:
Indeed, fixed!